### PR TITLE
[Ubuntu] Update npm parcel package to v2

### PR DESCRIPTION
**Pull request by V-Zabayrachny**
# Description
This PR updates the npm parcel package to v2

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Review by al-cheb — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Review by miketimofeev — CHANGES_REQUESTED** [review]
@V-Zabayrachny I couldn't find parcel in the readme file. I believe we have to add it to our software report.

File: images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1 [hunk 1]
```diff
@@ -68,6 +68,7 @@ $packageManagementList = @(
         (Get-NpmVersion),
         (Get-YarnVersion),
         (Get-PipxVersion),
+        (Get-ParcelVersion),
```
Comment (miketimofeev) [inline images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1:71]: [Parcel is a zero configuration build tool for the web](https://github.com/parcel-bundler/parcel), not the package management utility

Comment (V-Zabayrachny) [reply images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1]: Thanks. I fixed it

**Review by al-cheb — APPROVED** [review]

**Review by vsafonkin — APPROVED** [review]

**Review by dmitry-shibanov — APPROVED** [review]

**Review by miketimofeev — APPROVED** [review]

**Pull request merged**